test/compflags/albrecht/chplenv/chplenv.chpl

```diff
@@ -12,13 +10,11 @@ writeln("CHPL_LAUNCHER=",CHPL_LAUNCHER);
 writeln("CHPL_TIMERS=",CHPL_TIMERS);
 writeln("CHPL_UNWIND=",CHPL_UNWIND);
 writeln("CHPL_MEM=",CHPL_MEM);
-writeln("CHPL_JEMALLOC=",CHPL_JEMALLOC);
 writeln("CHPL_MAKE=",CHPL_MAKE);
 writeln("CHPL_ATOMICS=",CHPL_ATOMICS);
 writeln("CHPL_NETWORK_ATOMICS=",CHPL_NETWORK_ATOMICS);
 writeln("CHPL_GMP=",CHPL_GMP);
 writeln("CHPL_HWLOC=",CHPL_HWLOC);
 writeln("CHPL_REGEXP=",CHPL_REGEXP);
-writeln("CHPL_LLVM=",CHPL_LLVM);
```

Comment:
Removed since now by default we don't print any compile vars, even for --simple, right?

Reply:
Right.